README.md

```diff
@@ -1,353 +1,165 @@
 # website
 
 Site web du projet :)
-Nomalement on a une synchro automatique de la branche main sur le serveur.
 
 
-# Start the website
+# Start the project
 
-In order to deploy the website make sure to :
+## Using docker
 
-1. Start the docker (or install all required packets)
-2. Create ``/src/.env`` file (inspired by ``/src/.env.example``).
-    - Make sure ``APP_DEBUG=false`` (in case of production)
-    - Make sure to change DB informations
-    - Make sure to change ``DEFAULT_EMAIL = "test@test.com"`` and ``
-      DEFAULT_PASSWORD ="test123456"`` (it will be the first user loging and password)
-3. Run these lignes in ``src``
+### Using portainer on your machine
 
-```
-npm update
-composer update
-chmod -R 775 storage
-php artisan migrate --seed
-```
-
-# Connection à l'API
+You can go to [https://portainer.netw4ppl.tech](https://portainer.netw4ppl.tech) to manage the docker server but also
+your machine locally, provided you register your machine.
```

Comment:
"provided you register your machine" : is it like "providing you the ability to register your machine" ?

Reply:
yes